lib/utils.dart

```diff
@@ -6,24 +6,21 @@ import 'package:bars_frontend/main.dart';
 import 'package:flutter/material.dart';
 import 'package:http/http.dart' as http;
 
-class Pair {
-  dynamic first;
-  dynamic second;
-
-  Pair(this.first, this.second);
-}
-
 readJSON(String path) async {
-  String JSON = await rootBundle.loadString(path);
-  return jsonDecode(JSON);
+  String json = await rootBundle.loadString(path);
+  return jsonDecode(json);
 }
 
+/// Reads models, features and labels configs.
 readData() async {
-  Map<String, dynamic> serverConfig = await readJSON('assets/server.conf'); // TODO (far out): gather all this info not from file but thru GUI input
+  Map<String, dynamic> serverConfig = await readJSON(
+      'assets/server.conf'); // TODO (far out): gather all this info not from file but thru GUI input
```

Comment:
kill the TODO?

Reply:
yes, although this is still something that would be really nice to have this is the wrong place to talk about